src/idea.js

```diff
@@ -10,18 +10,18 @@ class Idea {
     localStorage.setItem("saved-ideas", JSON.stringify(allSavedIdeas));
   }
 
-  deleteFromStorage() {
-    var savedIdeas = JSON.parse(localStorage.getItem("saved-ideas"));
-    for (var i = 0; i < savedIdeas.length; i++){
-      if (savedIdeas[i].id === this.id) {
-        savedIdeas.splice(i, 1);
-      }
-    }
-    localStorage.setItem("saved-ideas", JSON.stringify(savedIdeas))
-  }
-
-  updateIdea() {
-    this.star = !this.star;
-    // this.star ? this.src = "images/star-active.svg" : this.src = "images/star.svg"
-  }
-}
+//   deleteFromStorage() {
+//     var savedIdeas = JSON.parse(localStorage.getItem("saved-ideas"));
+//     for (var i = 0; i < savedIdeas.length; i++){
+//       if (savedIdeas[i].id === this.id) {
+//         savedIdeas.splice(i, 1);
+//       }
+//     }
+//     localStorage.setItem("saved-ideas", JSON.stringify(savedIdeas))
+//   }
+//
+//   updateIdea() {
+//     this.star = !this.star;
+//     // this.star ? this.src = "images/star-active.svg" : this.src = "images/star.svg"
+//   }
+// }
```

Comment:
Be careful with temporary commenting, this currently breaks the syntax of this class.